syntax = "proto3";

option
java_package = "com.huasheng.quant.open.sdk.protobuf.common.response";
option java_outer_classname = "CommonBoolResponseProto";
message CommonBoolResponse {
	bool success = 1;
}